src/arx7.js

```diff
@@ -267,4 +289,14 @@ export class ARX7 {
       log(`ERROR: ${message.command}`);
     }
   }
+
+  moduleList() {
+    let list = '';
+
+    this.commands.forEach(command => {
+      list += command.constructor.name.toLowerCase() + ', ';
+    });
+
+    return list.substring(0, list.length - 2);
```

Comment:
This is kinda gross, and empty lists would make this weird.

maybe something like 

```javascript
let list = this.commands.map(command => { command.constructor.name.toLowerCase() });
return list.join(", ");
```